api/v2.0/legacy_swagger.yaml

```diff
@@ -3459,7 +3497,11 @@ definitions:
         description: The default storage quota for the new created projects.
       token_expiration:
         $ref: '#/definitions/IntegerConfigItem'
-        description: 'The expiration time of the token for internal Registry, in minutes.'
+        description: 'The expiration time of the token for internal Registry, in
+        minutes.'
+      robot_token_duration:
+        $ref: '#/definitions/IntegerConfigItem'
+        description: 'The expiration time of the robot account, in minutes.'
```

Comment:
Please change `in minutes` to`in days` in harbor 2.2

Reply:
updated.